lib/fizzy/execute.cpp

```diff
@@ -146,6 +146,14 @@ inline uint64_t popcnt64(uint64_t value) noexcept
 {
     return static_cast<uint64_t>(__builtin_popcountll(value));
 }
+
+struct label_context
```

Comment:
Maybe keep types at the top of the unnamed namespace (not to mix types with functions)